app-doc/heirloom-doctools/heirloom-doctools-160308.ebuild

```diff
@@ -0,0 +1,215 @@
+# Copyright 1999-2009 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: /var/cvsroot/gentoo-x86/app-doc/heirloom-doctools/Attic/heirloom-doctools-080407-r2.ebuild,v 1.3 2011/01/20 04:42:43 flameeyes dead $
+# revived from n-t-roff github fork.
+
+EAPI=6
+
+inherit flag-o-matic toolchain-funcs multilib
+
+DESCRIPTION="Classic Unix documentation tools ported from OpenSolaris"
+HOMEPAGE="http://heirloom.sourceforge.net/doctools.html"
```

Comment:
If you are using a fork, use the homepage of the fork. Do not pretend it's the original.